apps/front-client/src/features/sidebar/hooks/useUpdateUser.ts

```diff
@@ -0,0 +1,41 @@
+import { useMutation } from '@tanstack/react-query';
+import { User } from 'next-auth';
+import { useSession } from 'next-auth/react';
+
+export interface UpdateUserDto {
+  first_name: string;
+  last_name: string;
+  email: string;
+}
+
+const updateUser = async (
+  id: string,
+  updateUserDto: UpdateUserDto,
+  access_token?: string
+): Promise<User> => {
+  const response = await fetch(`${process.env.API_URL}/user/${id}`, {
+    method: 'PATCH',
+    headers: {
+      Authorization: `Bearer ${access_token}`,
```

Comment:
Il manque le Content-Type je pense que la CORS error vient de là, autant un POST il est capable de le deviner autant sur un PATCH je pense pas.
```suggestion
      Authorization: `Bearer ${access_token}`,
      'Content-Type': 'application/json',
```

Reply:
Ce n'est pas ça, il y était avant, marche pas non plus